daemon/logger/awslogs/cloudwatchlogs_test.go

```diff
@@ -579,7 +611,7 @@ func TestCollectBatchSimple(t *testing.T) {
 	}
 	d := make(chan bool)
 	close(d)
-	go stream.collectBatch(d)
+	go stream.collectBatch(context.TODO(), d)
```

Comment:
For gorountine-starting code, how about using context.WithCancel instead? So we could cancel the goroutine at the end.

Reply:
Yep